src/lua/InputApi.cpp

```diff
@@ -232,17 +236,94 @@ int LuaContext::input_api_is_mouse_button_released(lua_State* l) {
 int LuaContext::input_api_get_mouse_position(lua_State* l) {
 
   return LuaTools::exception_boundary_handle(l, [&] {
-    Point mouse_xy;
-    if (!InputEvent::get_global_mouse_position(mouse_xy)) {
+    Point mouse_xy = InputEvent::get_global_mouse_position();
+
+    lua_pushinteger(l, mouse_xy.x);
+    lua_pushinteger(l, mouse_xy.y);
+    return 2;
+  });
+}
+
+/**
+ * \brief Implementation of sol.input.is_finger_pressed().
+ * \param l The Lua context that is calling this function.
+ * \return Number of values to return to Lua.
+ */
+int LuaContext::input_api_is_finger_pressed(lua_State* l) {
+
+  return LuaTools::exception_boundary_handle(l, [&] {
+    int finger_id = LuaTools::check_int(l, 1);
+
+    if (finger_id == -1) {
```

Comment:
Hum sorry I forgot to remove that, actually there shouldn't be any forbidden value :)